onnxscript/ir/_core.py

```diff
@@ -882,35 +882,59 @@ class OptionalType(_RecursiveTypeBase):
 
 
 class Value(_protocols.ValueProtocol, _display.PrettyPrintable):
-    """IR Value."""
+    """IR Value.
+
+    A value is a named entity that can be used to represent an input or output of a graph,
+    a function, or a node. The information it stores corresponds to ``ValueInfoProto``
```

Comment:
Maybe more accurate to say that it generalizes `ValueInfoProto` ... it does store more info (producer, consumer, whether it is a constant, etc. etc.)

Reply:
Done